docs/managed-victoriametrics/alertmanager-setup-for-deployment.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Alertmanager and VMAlert configuration for Deployment
+aliases:
+- /managed-victoriametrics/alertmanager-configuration.html
+---
+
+## Alerting stack configuration and Managed VictoriaMetrics
+
+ Managed VictoriaMetrics supports configuration for alerting rules and notifications for it with alertmanager.
```

Comment:
Managed VictoriaMetrics supports configuring alerting rules and notifications through Alertmanager and internal vmalert.